modules/claims_api/app/services/claims_api/unsynchronized_evss_claims_service.rb

```diff
@@ -25,6 +25,19 @@ def all
       end.flatten
     end
 
+    def count
+      raw_claims = client.all_claims.body
+      EVSS_CLAIM_KEYS.each_with_object([]) do |key, claim_accum|
+        next unless raw_claims[key]
+
+        claim_accum << raw_claims[key].map do |raw_claim|
+          raw_claim['id']
+        end
+      end.flatten.count
```

Comment:
instead of this you should just be able to do something like:
```
EVSS_CLAIM_KEYS.reduce(0) { |sum, key| sum + raw_claims[key].count }
```

Reply:
implemented this instead 
```
EVSS_CLAIM_KEYS.reduce(0) do |sum, key| 
        next unless raw_claims[key]
        sum + raw_claims[key].count
      end
```